patches/export_pdf.patch

```diff
@@ -0,0 +1,97 @@
+diff --git a/pdf/pdf.cc b/pdf/pdf.cc
+index 7aba412..f57902a 100644
+--- a/pdf/pdf.cc
++++ b/pdf/pdf.cc
+@@ -81,7 +81,13 @@ const void* PPP_GetInterface(const char* interface_name) {
+   return pp::Module::Get()->GetPluginInterface(interface_name);
+ }
+ 
++}  // namespace chrome_pdf
++
++
++extern "C" {
++
+ #if defined(OS_WIN)
++PDF_EXPORT
+ bool RenderPDFPageToDC(const void* pdf_buffer,
+                                  int buffer_size,
+                                  int page_number,
+@@ -118,6 +124,7 @@ bool RenderPDFPageToDC(const void* pdf_buffer,
+ 
+ #endif  // OS_WIN
+ 
++PDF_EXPORT
+ bool GetPDFDocInfo(const void* pdf_buffer,
+                    int buffer_size, int* page_count,
+                    double* max_page_width) {
+@@ -176,4 +183,4 @@ bool RenderPDFPageToBitmap(const void* pdf_buffer,
+   return ret;
+ }
+ 
+-}  // namespace chrome_pdf
++}
+diff --git a/pdf/pdf.gyp b/pdf/pdf.gyp
+index f47e33a..6e0a12d 100644
+--- a/pdf/pdf.gyp
++++ b/pdf/pdf.gyp
+@@ -6,7 +6,7 @@
+   'targets': [
+     {
+       'target_name': 'pdf',
+-      'type': 'static_library',
++      'type': 'loadable_module',
+       'dependencies': [
+         '../base/base.gyp:base',
+         '../components/components.gyp:ui_zoom',
+@@ -17,6 +17,9 @@
+         '../third_party/pdfium/pdfium.gyp:pdfium',
+       ],
+       'ldflags': [ '-L<(PRODUCT_DIR)',],
++      'defines': [
++        'PDF_IMPLEMENTATION',
++      ],
+       'sources': [
+         'button.h',
+         'button.cc',
+diff --git a/pdf/pdf.h b/pdf/pdf.h
+index 37e72e5..dde2c13 100644
+--- a/pdf/pdf.h
++++ b/pdf/pdf.h
+@@ -5,6 +5,20 @@
+ #ifndef PDF_PDF_H_
+ #define PDF_PDF_H_
+ 
++#if defined(WIN32)
++
++#if defined(PDF_IMPLEMENTATION)
++#define PDF_EXPORT __declspec(dllexport)
```

Comment:
Since `PDF_EXPORT` is only used in `pdf.cc` and it is only compiled on Windows, I think we can use `__declspec(dllexport)` directly instead of defining `PDF_EXPORT`?